common/src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -29,6 +29,7 @@
     // Maximum interval between reports
     private static final long DEFAULT_CLOCK_STATE_INTERVAL_MILLIS = 500;
     private static final int DEFAULT_REPORT_TIMEOUT_MILLIS = 10 * 1000;
+    private static final int DEFAULT_CLIENT_RESET_INTERVAL_MILLIS = 5 * 60 * 1000; // 5 min
```

Comment:
I haven't thought about whether Android libraries want this feature, or not. (Maybe it will hurt battery performance?)